packages/components/src/components/DiagramFlow.vue

```diff
@@ -1,122 +0,0 @@
-<template v-slot:diagram>
-  <div class="diagram-flow" :key="renderKey" />
-</template>
-
-<script>
-import { CallTree, Event } from '@appland/models';
-import { FlowView, panToNode } from '@appland/diagrams';
```

Comment:
You should also remove `panToNode` function – it appears unused now. (And then `getParentOffset` from `util`, too.)